Sources/SwiftSyntaxBuilder/DeclBuildables.swift

```diff
@@ -159,7 +159,7 @@ public struct Struct: DeclBuildable {
   }
 
   public func buildDecl(format: Format, leadingTrivia: Trivia) -> DeclSyntax {
-    let structKeyword = Tokens.struct.withLeadingTrivia(leadingTrivia)
+    let structKeyword = Tokens.structKeyword.withLeadingTrivia(leadingTrivia)
```

Comment:
Should the new generated one have the `keyword` removed or is this good?

Reply:
I also think it is some verbose, but I didn't know what you preferred. 
I tried to se if there was a way to lower case first letter in `${token.name}`

Reply:
You can do it similar to how `swift_kind` is computed in [Token.py](https://github.com/apple/swift/blob/main/utils/gyb_syntax_support/Token.py#L24):

```swift
from gyb_syntax_support/kinds import lowercase_first_word

// ...

static let `${lowercase_first_word(token.name)}` = SyntaxFactory.make${token.name}Keyword().withTrailingTrivia(.spaces(1))
```

(or something like this, I haven’t tested the code)